lang/po/Firmware_pl.po

```diff
@@ -1654,7 +1655,7 @@ msgstr "Druk przerwany"
 #: ../../Firmware/messages.cpp:36 ../../Firmware/ultralcd.cpp:1080
 #: ../../Firmware/ultralcd.cpp:6879
 msgid "Print fan:"
-msgstr "WentWydruk:"
+msgstr "WentDruku:"
```

Comment:
`Went. Druku:` similar to `Went. hotend:

Reply:
Updated that and added few more fixes/improvements.